packages/emotion-primitives/src/index.js

```diff
@@ -0,0 +1,3 @@
+import { emotionPrimitive } from './emotionPrimitive'
```

Comment:
u can shorten it to just
```js
export { emotionPrimitive } from './emotionPrimitive'
```

Reply:
also it seems that u should export a default here (instead of named export) with
```js
export { emotionPrimitive as default } from './emotionPrimitive'
```